aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -292,6 +292,51 @@ module aptos_framework::account {
         account_resource.authentication_key = new_auth_key;
     }
 
+    public entry fun offer_rotation_capability(
+        account: &signer,
+        rotation_capability_sig_bytes: vector<u8>,
+        account_scheme: u8,
+        account_public_key_bytes: vector<u8>,
+        recipient_address: address,
+    ) acquires Account {
+        let addr = signer::address_of(account);
+        assert!(exists_at(addr) && exists_at(recipient_address), error::not_found(EACCOUNT_DOES_NOT_EXIST));
+
+        let account_resource = borrow_global_mut<Account>(addr);
+
+        // proof that this account intends to delegate its rotation capability to another account
+        let proof_challenge = RotationCapabilityOfferProofChallenge {
+            sequence_number: account_resource.sequence_number,
+            recipient_address,
+        };
+
+        // verify that the `SignerCapabilityOfferProofChallenge` is correct and signed by the account owner's private key
```

Comment:
Typo: SignerCapabilityOfferProofChallenge should be RotationCapabilityOfferProofChallenge

Reply:
Technically, you don't "verify correctness of RotationCapabilityOfferProofChallenge" here. You've just built the correct one above, so I would rephrase this as: "verify the signature on RotationCapabilityOfferProofChallenge by the account owner"

Reply:
ok will update the comment! good point